lambda-function/tests/unit/utils/mailservice.test.ts

```diff
@@ -0,0 +1,48 @@
+import MailService from "../../../src/utils/mailservice";
+import SMTPTransport from "nodemailer/lib/smtp-transport";
+
+jest.mock("nodemailer");
+jest.mock("nodemailer/lib/smtp-transport");
+
+const nodemailer = require("nodemailer"); //doesn't work with import. idk why
+const smtpTransport = require("nodemailer/lib/smtp-transport");
+
+describe("SMTP test", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it("Calls SMTPClient", async () => {
+    const sendMailMock = jest.fn().mockReturnValue({ someObjectProperty: 42 }); //we're not inspecting the response, so the mock return doesn't matter
+
+    nodemailer.createTransport.mockReturnValue({ sendMail: sendMailMock, verify: () => {} });
+
+    const mailService: MailService = new MailService();
+    const info: SMTPTransport.SentMessageInfo = await mailService.sendMail("dev@example.com", "Error Message");
+    expect(info).toBeDefined();
+
+    expect(sendMailMock).toHaveBeenCalled();
+  });
+
+  it.skip("Calls SMTPClient, transporter verify returns an error", async () => {
```

Comment:
Is this skip expected?

Reply:
Yeah.. for the time being.  Cleaning up in next story